polaris-react/src/components/Box/Box.tsx

```diff
@@ -0,0 +1,263 @@
+      top: borderTop ? borderTop : border,
+    } as Border;
+
+    const borderRadiuses = {
+      bottomLeft: borderRadiusBottomLeft
+        ? borderRadiusBottomLeft
+        : borderRadius,
+      bottomRight: borderRadiusBottomRight
+        ? borderRadiusBottomRight
+        : borderRadius,
+      topLeft: borderRadiusTopLeft ? borderRadiusTopLeft : borderRadius,
+      topRight: borderRadiusTopRight ? borderRadiusTopRight : borderRadius,
+    } as BorderRadius;
+
+    const margins = {
+      bottom: marginBottom ? marginBottom : margin,
+      left: marginLeft ? marginLeft : margin,
+      right: marginRight ? marginRight : margin,
+      top: marginTop ? marginTop : margin,
+    } as Spacing;
+
+    const paddings = {
+      bottom: paddingBottom ? paddingBottom : padding,
+      left: paddingLeft ? paddingLeft : padding,
+      right: paddingRight ? paddingRight : padding,
+      top: paddingTop ? paddingTop : padding,
+    } as Spacing;
+
+    const style = {
+      '--pc-box-margin-bottom': margins.bottom
+        ? `var(--p-space-${margins.bottom})`
+        : '',
+      '--pc-box-margin-left': margins.left
+        ? `var(--p-space-${margins.left})`
+        : '',
+      '--pc-box-margin-right': margins.right
+        ? `var(--p-space-${margins.right})`
+        : '',
+      '--pc-box-margin-top': margins.top ? `var(--p-space-${margins.top})` : '',
+      '--pc-box-padding-bottom': paddings.bottom
+        ? `var(--p-space-${paddings.bottom})`
+        : '',
+      '--pc-box-padding-left': paddings.left
+        ? `var(--p-space-${paddings.left})`
+        : '',
+      '--pc-box-padding-right': paddings.right
+        ? `var(--p-space-${paddings.right})`
+        : '',
+      '--pc-box-padding-top': paddings.top
+        ? `var(--p-space-${paddings.top})`
+        : '',
+      '--pc-box-border-bottom': borders.bottom
+        ? `var(--p-border-${borders.bottom})`
+        : '',
+      '--pc-box-border-left': borders.left
+        ? `var(--p-border-${borders.left})`
+        : '',
+      '--pc-box-border-right': borders.right
+        ? `var(--p-border-${borders.right})`
+        : '',
+      '--pc-box-border-top': borders.top
+        ? `var(--p-border-${borders.top})`
+        : '',
+      '--pc-box-border-radius-bottom-left': borderRadiuses.bottomLeft
+        ? `var(--p-border-${borderRadiuses.bottomLeft})`
+        : '',
+      '--pc-box-border-radius-bottom-right': borderRadiuses.bottomRight
+        ? `var(--p-border-${borderRadiuses.bottomRight})`
+        : '',
+      '--pc-box-border-radius-top-left': borderRadiuses.topLeft
+        ? `var(--p-border-${borderRadiuses.topLeft})`
+        : '',
+      '--pc-box-border-radius-top-right': borderRadiuses.topRight
+        ? `var(--p-border-${borderRadiuses.topRight})`
+        : '',
+      '--pc-box-shadow': shadow ? `var(--p-shadow-${shadow})` : '',
+    } as React.CSSProperties;
+
+    const className = classNames(styles.root, background && styles[background]);
```

Comment:
`background && styles[background]`

Can we just assign the background color to a custom property instead of creating classes for each?